libs/ui/layout/src/lib/interactive-table/interactive-table-column/interactive-table-column.component.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ContentChild,
+  EventEmitter,
+  Input,
+  Output,
+  TemplateRef,
+} from '@angular/core'
+import { CommonModule } from '@angular/common'
+
+@Component({
+  selector: 'gn-ui-interactive-table-column',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './interactive-table-column.component.html',
+  styleUrls: ['./interactive-table-column.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class InteractiveTableColumnComponent {
+  @ContentChild('header') header: TemplateRef<unknown>
+  @ContentChild('cell') cell: TemplateRef<unknown>
```

Comment:
I couldn't find in the review where those objects header and cell are used, what's their purpose?

Reply:
yes exactly! I think it was a good way to give the possibility to render anything in each column using templates